### PR TITLE
Completed german translation

### PR DESCRIPTION
**Pull request by oxivanisher**

### CODE REVIEW

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -319,10 +349,22 @@ msgid ""
 "        cooperate, then we will take it further.\n"
 "    "
 msgstr ""
+"\n"
+"        Du kannst mit dem Besitzers des Mudells\n"
```
Comment (winged) [inline locale/de/LC_MESSAGES/django.po]: Mudells?

**Review by winged — CHANGES_REQUESTED** [review]
Generally, I would use the respect-form for finders (and possibly for everybody else, too!) I know informal language is the norm amongst us RC pilots, but somebody who finds a model probably doesn't share that sentiment. Thus I'd use formal language here

File: locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -43,30 +43,45 @@ msgid ""
 "    <p>Here's a rough outline of what we're trying to build:\n"
 "    "
 msgstr ""
+"\n"
+"    <p>Wir sind ein paar Typen aus der Schweiz mit der Idee ein System zu "
```
Comment (winged) [inline locale/de/LC_MESSAGES/django.po]: "Typen" is extremely informal in german, not comparable to "Guys" in English. I suggest "Leute" instead

File: locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -43,30 +43,45 @@ msgid ""
...
+"(Möglich als Dienstleistung gegen eine kleine Gebühr)"
 
 #: modelreg/templates/about.html:13
 msgid ""
 "The data will be fully encrypted. Part of the QR code URL would be a random "
 "key to decrypt part of your information (for example phone number and name)"
 msgstr ""
+"Alle Daten sind voll verschlüsselt. Ein Teil des QR-Codes ist ein zufällig "
+"generierter Schlüssel, welcher zum entschlüsseln der Daten benötigt wird "
+"(wie zum Beispiel die Telefonnummer und der Name)"
```
Comment (winged) [inline locale/de/LC_MESSAGES/django.po:76]: Unrelated to the translation - Maybe we should fix this text as long as this is not yet implemented..

File: locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -43,30 +43,45 @@ msgid ""
...
 
 #: modelreg/templates/about.html:14
 msgid ""
 "Access via QR code will also give you the ability to notify the owner if you "
 "found a model"
 msgstr ""
+"Zugang zum QR-Code ermöglicht auch, dass du informiert wirst, wenn jemand "
+"dein Modell gefunden hat"
```
Comment (winged) [inline locale/de/LC_MESSAGES/django.po]: This is wrong - "Zugang *über den* QR-Code" would be better

File: locale/de/LC_MESSAGES/django.po (hunk 3 above)
Comment (oxivanisher) [reply locale/de/LC_MESSAGES/django.po:76]: Yes, I thought about that too. But to get the translation going, I just translated the English text.

Comment (winged) [reply locale/de/LC_MESSAGES/django.po:76]: Let's fix that in a separate issue/PR then.

Comment (winged) [reply locale/de/LC_MESSAGES/django.po:76]: See #43

**Review by winged — APPROVED** [review]
Looks good now

**Pull request merged**